VultisigApp/VultisigApp/Services/CryptoPriceService.swift

```diff
@@ -12,6 +12,10 @@ public class CryptoPriceService: ObservableObject {
     
     func getPrice(priceProviderId: String) async -> Double {
         
+        if priceProviderId.isEmpty {
+            return Double.zero
```

Comment:
priceProviderId should not be empty for correct app work. What is the cases when I can be empty?

Reply:
New tokens or community tokes are not listed on Coingecko yet or maybe only on CMC, so we can get the price by the pool pair using the same API